README.md

```diff
@@ -174,7 +176,7 @@ def createRaygunNotifyDeploymentTask(token,key,groupName,version,userName,userEm
 
 This function gets called from within the ```android {...}``` block of the Gradle file at each build in Android Studio and creates the appropriate parameterised task to notify the Raygun backend of your app's deployment.
 
-## Advanced features
```

Comment:
We mention Advanced features in most of our provider's documentation. For consistency and familiarity across the providers, we should keep this in place

Reply:
Fixed